dkc/core/views.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.admin.views.decorators import staff_member_required
+from django.db import connection
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+
+TREES_BY_FILE_COUNT_Q = """
+SELECT folder.id, folder.name, sq.c as file_count, pg_size_pretty(folder.size) as size
```

Comment:
I do not think it's essential to use raw SQL. Running two queries to render an internal dashboard seems far more reasonable.

Reply:
Cool, your assignment is to tell me what those two queries are and how to run them 👍